src/geometry/PolySet.cc

```diff
@@ -90,6 +90,11 @@ void PolySet::append_poly(const Polygon& poly)
   polygons.push_back(poly);
   this->dirty = true;
 }
+void PolySet::append_poly(bool marked) {
+  Polygon poly = Polygon();
+  poly.marked = marked;
```

Comment:
We should document somewhere what "marked" means. Also keep in mind that non-CGAL geometry evaluators may not have this feature, so the usefulness of it could be short-lived.